.github/workflows/alert.yml

```diff
@@ -0,0 +1,42 @@
+name: Alert
+on:
+  schedule:
+    - cron: '30 10 * * 1'
+  workflow_dispatch:
+jobs:
+  Alert:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.12
+      - name: Install pipx
+        run: python -m pip install pipx
+      - name: Install gha-token
+        run: python -m pipx install "git+https://github.com/hypothesis/gha-token.git"
+      - name: Install dependabot-alerts
+        run: python -m pipx install "git+https://github.com/hypothesis/dependabot-alerts.git"
```

Comment:
FIXME: This is installing `dependabot-alerts` from the `main` branch. Instead it should check out the repo and run `pipx install -e .`. That way if you run the workflow from a branch it'll use the code from that branch